File: 007+008/func.rb
def ham_so_bac_nhat(a, b)
  x = -(b/a)
  puts "x = #{'%.2f' %x}"
end

def ham_so_bac_hai(a, b, c)
  delta = b**2 - 4*a*c
  x1 = (-b - Math.sqrt(delta))/(2*a)
  puts "x1 = #{'%.5f' %x1}"
  x2 = (-b + Math.sqrt(delta))/(2*a)
  puts "x2 = #{'%.5f' %x2}"
end

File: 007+008/func.py
def ham_so_bac_nhat(a, b):
    print(f"x = {-(b/a)}")

def ham_so_bac_hai(a, b, c):
    Delta = b**2 - 4*a*c
    x1 = (-b - Delta**0.5)/(2*a)
    print (f"x1 = {"{:.5f}".format(x1)}")
    x2 = (-b + Delta**0.5)/(2*a)
    print (f"x2 = {"{:.5f}".format(x2)}")
